wasmer.toml

```
@@ -14,5 +14,10 @@ abi = "wasi"
 [[command]]
 name = "server"
 runner = "https://webc.org/runner/wcgi"
-module = "wcgi"
-annotations = { wcgi = { dialect = "rfc-3875" } }
+module = "server"
+
+[command.annotations]
+wcgi = { dialect = "rfc-3875" }
+
+[command.annotations.wasi]
+env = ['SCRIPT_NAME=rust_wcgi']
```

Would it make more sense for users if we had tables for both sets of annotations?

```suggestion
[command.annotations.wcgi]
dialect = "rfc-3875"

[command.annotations.wasi]
env = ['SCRIPT_NAME=rust_wcgi']
```

Or maybe we should inline them both?

```suggestion
[command.annotations]
wcgi = { dialect = "rfc-3875" }
wasi = { env = ['SCRIPT_NAME=rust_wcgi'] }
```

@dynamite-bud  I'll leave it up to you to decide which looks cleaner.

---

Well actually `wasmer app create` automatically formatted it